### PR TITLE
CgManifest auto update method

### PR DESCRIPTION
**Pull request by olmisa**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
A new method to update commit hashes in cgmanifest was added to version.cake file.

## Related PRs or issues

[AB#79268](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/79268)

### CODE REVIEW

File: scripts/update-cgmanifest.ps1 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (MatkovIvan) [inline scripts/update-cgmanifest.ps1:1]: Do not need to these wrapper scripts

File: version.cake [hunk 2]
```diff
@@ -15,8 +16,14 @@ using System.Text.RegularExpressions;
 using System.Xml.Linq;
 using System.Xml.XPath;
 
-var AndroidSdkRepoName = "microsoft/appcenter-sdk-android";
-var AppleSdkRepoName = "microsoft/appcenter-sdk-apple";
+const string AndroidSdkRepoName = "microsoft/appcenter-sdk-android";
+const string AppleSdkRepoName = "microsoft/appcenter-sdk-apple";
+
+var ReposToUpdate = new List<string>()
```
Comment (MatkovIvan) [inline version.cake:22]: This can be just local var. Or even no need to this array at all

File: scripts/update-cgmanifest.ps1 (hunk 1 above)
Comment (olmisa) [reply scripts/update-cgmanifest.ps1]: Fixed

File: version.cake (hunk 2 above)
Comment (olmisa) [reply version.cake]: Fixed

File: version.cake [hunk 3]
```diff
@@ -404,4 +405,80 @@ public void UpdateConfigFileSdkVersion(string newVersion)
...
+    var repoUrl = gitData["repositoryUrl"].Value<string>();
+    var currentRepoName = repoUrl.Contains(AndroidSdkRepoName) ? AndroidSdkRepoName : AppleSdkRepoName;
+    var releaseTag = GetReleaseTag(currentRepoName);
+    if (string.IsNullOrEmpty(releaseTag))
+    {
+        Warning($"Repository url: {repoUrl}. Release tag '{releaseTag}' was not found.");
+        return;
+    }
+
+    var tagsRequest = CreateGitHubRequest($"repos/{currentRepoName}/tags");
```
Comment (annakocheshkova) [inline version.cake:459]: Move to const

Comment (olmisa) [reply version.cake]: Fixed

**Pull request merged**